DBZ-4128 Add Debezium Kafka Connect REST Extension to Debezium Kafka Connect NIGHTLY container image

rk3rn3r opened the pull request:
* minor cleanup

closes https://issues.redhat.com/browse/DBZ-4128

jpechane commented on connect/snapshot/Dockerfile:
```diff
@@ -11,17 +11,32 @@ ENV DEBEZIUM_VERSION=$DEBEZIUM_VERSION \
-    done
+done
+
+RUN for PACKAGE in {connect-rest-extension,}; do \
```
Would it be possible to merge the two `extension` loops? IIUC they only differ in the filename suffix.

rk3rn3r replied:
No, they also differ in what they evaluate from the `maven-metadata.xml`.

rk3rn3r commented on connect/snapshot/Dockerfile:
```diff
@@ -11,17 +11,32 @@ ENV DEBEZIUM_VERSION=$DEBEZIUM_VERSION \
-RUN for PACKAGE in {scripting}; do \
-    SNAPSHOT_VERSION=$(curl -fSL $MAVEN_OSS_SNAPSHOT/io/debezium/debezium-$PACKAGE/$DEBEZIUM_VERSION/maven-metadata.xml | awk -F'<[^>]+>' '/<extension>tar.gz<\/extension>/ {getline; print $2}'); \
-    curl -fSL -o /tmp/package.tar.gz \
+RUN for PACKAGE in {scripting,}; do \
```
yes, it was like that before. I guess good old premature optimization. that's the correct Bash syntax for a list and I only added the missing comma, else Bash would handle it as the string `{scripting}` and not as list.

rk3rn3r commented:
I expected this to be already merged as I need the container image to finish the DBZ UI tests and next week Thursday is my last day before my PTO.
I was just adding some logging in a single file and fixed some really minor code mistakes (see my comment on Bash list vs string above and made curl really silent) beside the added lines for the actual task. If it would be 2 or more files and more than 4 changed lines (additionally to the lines I added for the task) I would have done different commits for sure!
